moto/databrew/models.py

```diff
@@ -0,0 +1,62 @@
+from collections import OrderedDict
+from datetime import datetime
+
+from moto.core import BaseBackend
+from moto.core import BaseModel
+from moto.utilities.paginator import paginate
+
+from .exceptions import RecipeAlreadyExistsException
+from .exceptions import RecipeNotFoundException
+
+
+class DataBrewBackend(BaseBackend):
+    PAGINATION_MODEL = {
+        "list_recipes": {
+            "input_token": "next_token",
+            "limit_key": "max_results",
+            "limit_default": 100,
+            "unique_attribute": "name",
+        },
+    }
+
+    def __init__(self):
+        self.recipes = OrderedDict()
+
+    def create_recipe(self, recipe_name, recipe_description, recipe_steps, tags):
+        # https://docs.aws.amazon.com/databrew/latest/dg/API_CreateRecipe.html
+        if recipe_name in self.recipes:
+            raise RecipeAlreadyExistsException()
```

Comment:
We should have a separate test for this scenario

Reply:
yeah, that makes sense. will add one.